cdc/owner.go

```diff
@@ -758,7 +765,7 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 			// A changefeed will not enter the map twice, because in run(),
 			// handleAdminJob() will always be executed before flushChangeFeedInfos(),
 			// ensuring that the previous changefeed in staleChangeFeeds has been stopped and removed from o.changeFeeds.
-			if changefeed.status.CheckpointTs < minGCSafePoint {
+			if minGCSafePoint != 0 && changefeed.status.CheckpointTs <= minGCSafePoint {
```

Comment:
We don't need to check `minGCSafePoint != 0` since `changefeed.status.CheckpointTs` is always larger than 0

Reply:
- We need to check `minGCSafePoint != 0` here because `o.pdGCSafePoint` may be `0` at the beginning, resulting in `minGCSafePoint` being `0` and will stop all changefeeds.  
- When the `minGCSafePoint` value is `0`, just skip the action to avoid the situation described above.

Reply:
please add a comment for checking `minGCSafePoint != 0` and `changefeed.status.CheckpointTs <= minGCSafePoint`

Reply:
done

Reply:
These checks here and below (line:799) are for almost the same reason.